integration/class-enable-mastodon-apps.php

```diff
@@ -127,8 +127,9 @@ public static function api_account_add_followers( $account, $user_id ) {
 			}
 		}
 
-		$account->acct = $user->get_webfinger();
+		$account->acct = $user->get_preferred_username();
```

Comment:
shouldn't this be a WebFinger acct?

Reply:
No, because it's a local account, it should not have a server name, otherwise your identity will show as `@user@server@server`.

Reply:
ah, I see!